config/image-optimize.php

```diff
@@ -1,75 +1,23 @@
 <?php
-/**
- * ImageOptimize plugin for Craft CMS 3.x
- *
- * Automatically optimize images after they've been transformed
- *
- * @link      https://nystudio107.com
- * @copyright Copyright (c) 2017 nystudio107
- */
 
-/**
- * ImageOptimize config.php
- *
- * This file exists only as a template for the ImageOptimize settings.
- * It does nothing on its own.
- *
- * Don't edit this file, instead copy it to 'craft/config' as
- * 'image-optimize.php' and make your changes there to override default
- * settings.
- *
- * Once copied to 'craft/config', this file will be multi-environment aware as
- * well, so you can have different settings groups for each environment, just
- * as
- * you do for 'general.php'
- */
+use nystudio107\imageoptimize\imagetransforms\CraftImageTransform;
 
 return [
     '*' => [
-        //  What transform method should be used for image transforms?
-        'transformMethod' => 'craft',
-
-        // Domain for the Imgix transform service
-        'imgixDomain' => '',
-
-        // API key for the Imgix transform service
-        'imgixApiKey' => '',
-
-        // The optional security token used to sign image URLs from Imgix
-        'imgixSecurityToken' => '',
-
-        // Should the image variants in an Asset Volume be automatically re-saved when saving
-        // an OptimizedImages field, saving an Asset Volume that has an OptimizedImages field
-        // in its layout, or saving the ImageOptimized settings. Set this to false only if
-        // you will be manually using the CLI console command to resave image variants
+        'transformClass' => CraftImageTransform::class,
+        'imageTransformTypeSettings' => [],
         'automaticallyResaveImageVariants' => true,
-
-        // Should image variant be created on Asset save (aka BeforePageLoad)
-        'generateTransformsBeforePageLoad' => true,
-
-        // Set to false to disable all placeholder generation
+        'generateTransformsBeforePageLoad' => false,
```

Comment:
Wil deze zeggen dat de transforms niet meer via de queue gaan @Numkil ?

Reply:
Goed gezien, na bespreken hadden we besloten het wel via de queue te blijven laten lopen! Heb de code aangepast en op true gezet!